### ISSUE TITLE
[Feature request]: `sbb-title` should set `role` and `aria-level` as HTML attributes

### ISSUE THREAD

**Issue by georgwittberger-telekom-mms** [sbb-design-systems/lyne-components#4135]
### Preflight Checklist

- [x] I have read the [Contributing Guidelines](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [issue tracker](https://github.com/orgs/sbb-design-systems/projects/4) for an issue that matches the one I want to file, without success.

### Request type

Request for enhancement of a component

### It affects the following packages

Components

### Functionality

In former implementations of the `sbb-title` component it used to set its `role="heading"` and the respective `aria-level` attribute on the host element so that these values were actually reflected in the DOM.

A while ago this behavior has changed and these properties are only set internally. This works for screen readers but tools relying on the DOM cannot recognize these custom heading components now.

We have received an issue from the SEO team that their tools can no longer determine the headings in our page: https://flow.sbb.ch/browse/SBBCH-22727

Is it possible to get the previous behavior back, i.e. `role` and `aria-level` attributes are set as HTML attributes?

If not, what is the recommended solution to our SEO problem? Should we set these attributes on our own on the custom element?

**Comment by kyubisation** [comment]
Hello @georgwittberger-telekom-mms 
This change was intentional to prevent attributes from spawning on the host (especially as React does not like attributes it does not control).
As this is a tool problem, we will not change this again.

**Comment by kyubisation** [comment]
Sorry, I missed your last question.
You can technically set `role` and `aria-level` manually, if really required.
